properties/P000088.md

```diff
@@ -1,18 +1,25 @@
 ---
 uid: P000088
 name: Collectionwise normal
+aliases:
+  - CWN
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
   - wikipedia: Collectionwise_normal_space
     name: Collectionwise Normal Space on Wikipedia
   - zb: "0684.54001"
     name: General Topology (Engelking, 1989)
+  - zb: "0529.54017"
+    name: Collectionwise Hausdorff versus collectionwise normal with respect to compact sets (Reed)
 ---
 
 For every discrete family $(F_i)_{i \in I}$  of closed subsets of $X$ there exists a pairwise disjoint family of open sets $(U_i)_{i \in I}$, such that $F_i \subseteq U_i$.
```

Comment:
Would the following read better?

For every discrete family $(F_i)_{i \in I}$  of closed subsets of $X$, there exists a pairwise disjoint family of open sets $(U_i)_{i \in I}$ such that $F_i \subseteq U_i$ for all $i$.

Reply:
Definitely.  Making the change.